_includes/_code/lakes-interactive/lib/1st-try.dart

```diff
@@ -0,0 +1,138 @@
+import 'package:flutter/material.dart';
+// Uncomment lines 3 and 6 to view the visual layout at runtime.
+//import 'package:flutter/rendering.dart' show debugPaintSizeEnabled;
+
+void main() {
+  //debugPaintSizeEnabled = true;
+  runApp(new MyApp());
+}
+
+IconData icon = Icons.star;
+
+class FavoriteWidget extends StatefulWidget {
+  /**
+  new Icon(
+  icon,
+  color: Colors.red[500],
+  ),
+  new Text('41'),
+      */
```

Comment:
Sorry, Adam. The "1st try" code should NOT be part of this repo. It was one of my intermediate versions. I've deleted it.